predelete/predelete.go

```diff
@@ -62,6 +67,33 @@ func PreDeleteOperatorSecret(clientset *kubernetes.Clientset) error {
 	return nil
 }
 
+func PreDeleteEnterprisePortalSecret(clientset *kubernetes.Clientset) error {
+	fmt.Println("Running pre delete hook")
+	secrets, err := clientset.CoreV1().Secrets(os.Getenv("TYK_POD_NAMESPACE")).List(context.TODO(), metav1.ListOptions{})
```

Comment:
Same comment as before
```suggestion
	secrets, err := clientset.CoreV1().Secrets(os.Getenv(TykPodNamespaceEnvVar)).List(context.TODO(), metav1.ListOptions{})
```